docs/en/user_guides/inference.md

```diff
@@ -1,20 +1,20 @@
 # Inference with existing models
```

Comment:
How about changing to `Inference with trained models`?

Reply:
I think the range of existing models is broader, which includes trained models, pruned models, manually converted models, etc. All of them may run inference with MMPose.